src/logger/Cargo.toml

```diff
@@ -17,3 +17,6 @@ serde_json = "1.0.78"
 thiserror = "1.0.32"
 vm-superio = "0.7.0"
 utils = { path = "../utils" }
+
+[dev-dependencies]
+uuid = { version = "1.4.1", features=["v4"] }
```

Comment:
Is this dependency used? I don't see it anywhere?

Reply:
It's used in the test.